k8s/k8s.go

```diff
@@ -282,13 +281,12 @@ func CmdAddK8s(args *skel.CmdArgs, conf utils.NetConf, nodename string, calicoCl
 		utils.ReleaseIPAllocation(logger, conf.IPAM.Type, args.StdinData)
 		return nil, err
 	}
-	endpoint.Spec.MAC = &cnet.MAC{HardwareAddr: mac}
+	endpoint.Spec.MAC = mac.String()
 	endpoint.Spec.InterfaceName = hostVethName
-	endpoint.Metadata.ActiveInstanceID = args.ContainerID
```

Comment:
We still need to update the the ContainerID here!

Reply:
dunno what I was thinking, I moved it up in the `if` block, but I realize this would bring back the double ADD bug you recently fixed. I'll move it back to here